editor/src/components/canvas/canvas-strategies/strategies/shared-move-strategies-helpers.ts

```diff
@@ -189,18 +192,21 @@ export function applyMoveCommon(
         canvasState.scale,
       )
       const commandsForSelectedElements = getMoveCommands(snappedDragVector)
-      return strategyApplicationResult([
-        ...commandsForSelectedElements.commands,
-        updateHighlightedViews('mid-interaction', []),
-        setSnappingGuidelines('mid-interaction', guidelinesWithSnappingVector),
-        pushIntendedBoundsAndUpdateGroups(
-          commandsForSelectedElements.intendedBounds,
-          'starting-metadata',
-        ),
-        setElementsToRerenderCommand([...targets, ...targetsForSnapping]),
-        setCursorCommand(CSSCursor.Select),
-        setActiveFrames(getActiveFrames(commandsForSelectedElements.intendedBounds)),
-      ])
+      return strategyApplicationResult(
+        [
+          ...commandsForSelectedElements.commands,
+          updateHighlightedViews('mid-interaction', []),
+          setSnappingGuidelines('mid-interaction', guidelinesWithSnappingVector),
+          pushIntendedBoundsAndUpdateGroups(
+            commandsForSelectedElements.intendedBounds,
+            'starting-metadata',
+          ),
+
+          setCursorCommand(CSSCursor.Select),
+          setActiveFrames(getActiveFrames(commandsForSelectedElements.intendedBounds)),
+        ],
+        'rerender-all-elements',
```

Comment:
based on the diff it looks like this should be [...targets, ...targetsForSnapping]

Reply:
Fixed in 71214dfb2.